tests/08-native-runs/07-aes-ccm/Readme.md

```diff
@@ -0,0 +1,2 @@
+To generate test vectors, run:
+./generate-test-vectors.py > test-vectors.c
```

Comment:
Add
```
pip install pycryptodome
```
or something to resolve the dependency.

Reply:
Did that. Also, switched to python3

Reply:
👍 I confirmed the code actually was able to run with Python 2 and Python 3 :-)